### ISSUE TITLE
Issue in running any model

### ISSUE THREAD

**Issue by jenil0108** [ollama/ollama#3908]
### What is the issue?

Previously I was able to run llama2 normally. But ever since llama3 came i am not able to run llama2 or llama3. On performing ollama run llama2 i get the following error.

`Error: error starting the external llama server: exec: "ollama_llama_server": executable file not found in $PATH`

I looked through some previous issues which stated to `reinstall ollama`, restart ollama via `brew services restart ollama`.
Both give the same error.
I also tried running `ollama serve`, which gives the following error

`Error: listen tcp 127.0.0.1:11434: bind: address already in use.`

Had found the same issue on another bug report which was closed saying issue solved with merge. But still having the same issue. 
One suggestion i had seen was setting the ollama_path manually by `export OLLAMA_HOST = localhost: 8.8.8.8`.
After which i could perform `ollama serve`. 
But then when i do run the `ollama run llama2` on a separate window

I get the following error 
```
pulling manifest
Error: pull model manifest: file does not exist
```

But on doing `ollama list` i clearly see both the models there.
Also tried pulling the models again but still no progress.

I am using Macbook M3 Pro and cli i am using is iterm2.

### OS

macOS

### GPU

Apple

### CPU

Apple

### Ollama version

0.1.32

**Comment by jmorganca** [comment]
Hi there, this should be fixed as of 0.1.33+. Sorry about the issue!

**Comment by xianengqi** [comment]
Thanks for you. `brew services restart ollama` solved my problem.
I encounter problem is `Error: error starting the external llama server: fork/exec /var/folders/9r/67wg5p_10bsglfgr7trswysc0000gn/T/ollama1123600442/runners/metal/ollama_llama_server: no such file or directory`
but I run `brew services restart ollama` Done problem. Thanks~
